admin_guide/managing_projects.adoc

```diff
@@ -90,39 +90,69 @@ to all authenticated developers by default.
 
 [[disabling-self-provisioning]]
 === Disabling Self-provisioning
-Removing the `self-provisioners`
-xref:../architecture/additional_concepts/authorization.adoc#roles[cluster role]
-from authenticated user groups will deny permissions for self-provisioning any new projects.
 
-----
-$ oc adm policy remove-cluster-role-from-group self-provisioner system:authenticated system:authenticated:oauth
-----
-
-When disabling self-provisioning, set the `projectRequestMessage` parameter in the
-*_master-config.yaml_* file to instruct developers on how to request a new
-project. This parameter is a string that will be presented to the developer in
-the web console and command line when they attempt to self-provision a project.
-For example:
+You can prevent an authenticated user group from self-provisioning new projects.
 
+. Log in as a user with
+xref:../architecture/additional_concepts/authorization.adoc#roles[*cluster-admin*]
+privileges.
+. Remove the `self-provisioners`
+xref:../architecture/additional_concepts/authorization.adoc#roles[cluster role]
+from the group.
```

Comment:
Remove from what group? Or, is this obvious?

Reply:
I read it as `system:authenticated` as the group. @xltian, is that right?

Reply:
@barleyer, is `system:authenticated` the group here?

Reply:
@kalexand-rh ，according to `self-provisioners` clusterrolebinding file，it should be `system:authenticated:oauth`.